fpga_interchange/add_prim_lib.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+""" Utility for adding a primitive library to a device from Yosys JSON """
+import argparse
+import json
+import re
+
+from fpga_interchange.interchange_capnp import Interchange, read_capnp_file, write_capnp_file
+from fpga_interchange.logical_netlist import LogicalNetlist, Cell, \
+        CellInstance, Direction, Library
+
+from fpga_interchange.yosys_json import is_bus
+
+def get_prims():
```

Comment:
Function is unused, and looks like it wouldn't work anyways (undefined variable `libraries`)?

Reply:
Yes this was from an earlier attempt and is dead code, now removed